android/java/org/chromium/components/safe_browsing/BraveSafeBrowsingApiHandler.java

```diff
@@ -0,0 +1,161 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
We now have a check in CI that requires an HTTPS URL here: https://mozilla.org/MPL/2.0/

Reply:
(same with the files below)

Reply:
hm, the CI has passed and green.

Reply:
I rebased the branch today. I can update the above files though, but we will continue to have an ongoing problem as people just copy/paste the header from other files. We need to replace it everywhere.

Reply:
Hm, not sure when it's run, but there's a check in eslint: https://bravesoftware.slack.com/archives/C7VLGSR55/p1668083941517119

Reply:
There was a PR to update all existing files, maybe it hasn't fully landed yet. In any case, if it's not enforced right now, it will be in the near future and so I'd recommend at least fixing it in new files.

Reply:
that's been addressed